### DISCUSSION TITLE
Allow another `WHERE` instead of `AND`

### DISCUSSION THREAD

**Discussion by soerenwolfers** [duckdb/duckdb#13998] [Ideas]
Would it be possible to allow 
```sql
SELECT *
FROM (VALUES (1, 2), (3, 4)) t(a, b)
WHERE a > 0
WHERE b > 2
```
instead of 
```sql
SELECT *
FROM (VALUES (1, 2), (3, 4)) t(a, b)
WHERE a > 0
AND b > 2
```
?

In interactive exploration, I often copy / cut / paste / comment out filter conditions and it's annoying that I have to treat the first line (e.g. `WHERE a > 0` above) different from the other ones (e.g., `AND b > 2`). 

This is in some way equivalent to duckdb's great feature of allowing trailing commas in the select clause.

**Comment by frankzandhuis** [comment]
I often make use of the "trick" to do a `WHERE 1=1` as the first statement. So after that I can use only `AND` statements. Not sure if this has any downsides with respect to efficiency.

For your code:
```
SELECT *
FROM (VALUES (1, 2), (3, 4)) t(a, b)
WHERE 1=1
AND a > 0
AND b > 2
```
I love your idea though. Would be great if this could also be added for joins, for `ON` and `AND`.

**Reply by soerenwolfers** [reply]
yeah, I was using `WHERE true` but it's sad
